pytorch_lightning/trainer/trainer.py

```diff
@@ -2004,6 +2004,10 @@ def _results(self) -> Optional[ResultCollection]:
         if active_loop is not None:
             return active_loop._results
 
+    def should_exit_gracefully(self, status: str) -> None:
```

Comment:
any alternative names considered here? just `exit_gracefully`? the public api on the trainer also needs a docstring.

Reply:
Changed to `_exit_gracefully_on_signal`.